docs/storage/develop/index.md

```diff
@@ -0,0 +1,37 @@
+---
+sidebar_position: 6
+sidebar_label: Develop
+title: Developing applications with Momento SDKs
+description: Learn the core concepts for getting started with the Momento SDKs.
+---
+
+import { SdkExampleTabs } from "@site/src/components/SdkExampleTabs";
+// This import is necessary even though it looks like it's un-used; The inject-example-code-snippet
+// plugin will transform instances of SdkExampleTabs to SdkExampleTabsImpl
+import { SdkExampleTabsImpl } from "@site/src/components/SdkExampleTabsImpl";
+
+# Developing applications with Momento SDKs
+
+Welcome! This page provides information about common constructs you will need in order to assemble Momento clients in all of our SDKs. This page covers how to provide your Momento credentials (called API keys), how to configure your client, and some basic information about error handling and production readiness.
+
+## Constructing a Storage Client
+
+The `StorageClient` is the main object you will use in your code to interact with Momento services. To instantiate one, you need to pass a `CredentialProvider` and a `Configuration`.
+
+Here is an example of how to construct a `StorageClient`:
+
+<SdkExampleTabs snippetId={'API_Storage_InstantiateClient'} />
+
+## Instantiating credential providers using Momento API keys
+
+You need to provide a Momento API key when instantiating a Momento client. If you don't have one yet, you can get one from the [Momento Web Console](https://console.gomomento.com/). Once you have a token, provide it to Momento SDKs when you create an instance of `CredentialProvider`. There are convenient factory methods provided to construct a `CredentialProvider` object, either from an environment variable or from a String. Below is an example of how to instantiate `CredentialProvider` from an environment variable:
+
+<SdkExampleTabs snippetId={'API_CredentialProviderFromEnvVar'} />
```

Comment:
looks like the JS snippets are still using the old style of passing in objects: https://github.com/momentohq/client-sdk-javascript/blob/main/examples/nodejs/cache/doc-example-files/doc-examples-js-apis.ts#L113-L120

Reply:
ty, fixed in https://github.com/momentohq/client-sdk-javascript/pull/1388 but I think it's fine to let that trickle in to the docs whenever the build timing works out.